keras_rs/src/layers/embedding/jax/test_utils.py

```diff
@@ -275,8 +404,8 @@ def _compute_expected_lookup_grad(
     embedding_dim = activation_gradients.shape[1]
     sample_lengths = jnp.array([len(sample) for sample in samples.tokens])
     rows = jnp.repeat(jnp.arange(batch_size), sample_lengths)
-    cols = jnp.concatenate(jnp.unstack(samples.tokens))
-    vals = jnp.concatenate(jnp.unstack(samples.weights)).reshape(-1, 1)
+    cols = jnp.concatenate(np.unstack(samples.tokens))  # type: ignore[attr-defined]
```

Comment:
I really don't understand the mypy issue with this.  The mypy job passes locally with the exact same version of `numpy` and `mypy` installed.  The `np.unstack` attribute _does_ exist in numpy 2.2.6, which is what is installed by the CI.  I need `np.unstack` because the arrays are ragged, so `jnp.unstack` throws an error.

Reply:
Added `unused-ignore` to the `pyproject.toml` to prevent local failure.